skrifa/src/outline/mod.rs

```diff
@@ -582,6 +584,14 @@ impl<'a> OutlineGlyphCollection<'a> {
             Some((gid, glyph))
         })
     }
+
+    pub(crate) fn common(&self) -> Option<&OutlinesCommon<'a>> {
```

Comment:
Not new in this change, but perhaps a naming related question: What "common" is, is not quite clear to me. Is that documented somewhere, or could we pick a more descriptive name?

Reply:
Please bear with me while under construction :) This is a shim to resolve unexpected data dependencies for autohinting and is tracked for cleanup in #1085

Reply:
Ok with me to track this in #1085.